module-3/README.md

```diff
@@ -15,17 +15,16 @@ Now that you have a service deployed and a working CI/CD pipeline to deliver cha
 
 #### Create a DynamoDB Table
 
-To add a DynamoDB table to the architecture, we have included another JSON CLI input file that defines a table called **MysfitsTable**. This table will have a primary index defined by a hash key attribute called **MysfitId**, and two more secondary indexes.  The first secondary index will have the hash key of **GoodEvil** and a range key of **MysfitId**, and the other secondary index will have the hash key of **LawChaos** and a range key of **MysfitId**.  These two secondary indexes will allow us to execute queries against the table to retrieve all of the mysfits that match a given Species or Alignment to enable the filter functionality.  You can view this file at `/module-3/aws-cli/dynamodb-table.json`. No changes need to be made to this file and it is ready to execute.
+To add a DynamoDB table to the architecture, we have included another AWS CDK Application that defines a table called **MysfitsTable**. This table will have a primary index defined by a hash key attribute called **MysfitId**, and two more secondary indexes.  The first secondary index will have the hash key of **GoodEvil** and a range key of **MysfitId**, and the other secondary index will have the hash key of **LawChaos** and a range key of **MysfitId**.  These two secondary indexes will allow us to execute queries against the table to retrieve all of the mysfits that match a given Species or Alignment to enable the filter functionality.
 
-To create the table using the AWS CLI or PowerShell, execute the following command in your terminal in Visual Studio Code:
+To create the table using the AWS CDK, execute the following command in your terminal in Visual Studio Code:
 
-`Bash`
-```
-aws dynamodb create-table --cli-input-json file://module-3/aws-cli/dynamodb-table.json
-```
-`PowerShell`
+`AWS CDK`
 ```
-./module-3/ps1/CreateDDBTable.ps1
+cd ./module-3/cdk

```

Comment:
Same comment as above - it should be straightforward for people to write this CDK code. Ideally they should be able to just add this stack to their app.